src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import "./PostInfo.scss";
+import { CommentList } from "../CommentList";
+import { UserInfo } from "../UserInfo";
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        <UserInfo post={post} />
+      </p>
```

Comment:
Not fixed. NOTE: user is possible null, so you need to add check if user exist
![2023-08-31_15h18_34](https://github.com/mate-academy/react_static-list-of-posts-js/assets/61904995/77749c7d-13a6-48b6-bf68-5d40c14658ee)